### PR TITLE
input and image component completed.

### PR DESCRIPTION
**Pull request by yogesh-**
For Image:
- Square image
- round image
- responsive image

For Input:
- Form input
- Form input Error
- Radio Buttons
- Checkbox

https://user-images.githubusercontent.com/7193961/152552869-cc0127ff-3270-43d2-9ad6-0be98e9a12f3.mp4

### CODE REVIEW

File: css/input.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+.form-err :is(input,button,p){
+    margin: 1rem 0rem;
+}
+
+.radio input, .checkbox input{
+margin: 1rem 0rem 1rem 2rem;
+}
+
```
Comment (haarsh24) [inline css/input.css:26]: use Formatting

**Review by haarsh24 — APPROVED** [review]

File: documentation.html [hunk 2]
```diff
@@ -696,11 +696,274 @@ <h3>Rs.4900 <Span>Rs.9999</Span></h3>
...
+                            <img class="square-img" src="./images/sample1.jpg" alt="alt-image">
+                            <img class="square-img" src="./images/sample2.jpg" alt="alt-image">
+
+
+                    </div>
+
+                    <div class="flex-row code-snippet">
+                        <iframe
+                            src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=cobalt&wt=none&l=auto&width=873.5&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Fira+Code&fs=14px&lh=152%25&si=false&es=2x&wm=false&code=%2509%2509%2509%253Cimg%2520class%253D%2522square-img%2522%2520src%253D%2522.%252Fimages%252Fnike.png%2522%2520alt%253D%2522sneakers%2522%253E%250A%2509%2509%2509%253Cimg%2520class%253D%2522square-img%2522%2520src%253D%2522.%252Fimages%252Fsample1.jpg%2522%2520alt%253D%2522alt-image%2522%253E%250A%2509%2509%2509%253Cimg%2520class%253D%2522square-img%2522%2520src%253D%2522.%252Fimages%252Fsample2.jpg%2522%2520alt%253D%2522alt-image%2522%253E"
+                            style="width: 873px; height: 122px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (akash-1318) [inline documentation.html]: do not use inline styles.

File: documentation.html [hunk 3]
```diff
@@ -696,11 +696,274 @@ <h3>Rs.4900 <Span>Rs.9999</Span></h3>
...
+                            style="width: 873px; height: 398px; border:0; transform: scale(1); overflow:hidden;"
+                            sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div> 
+
+
+                </div>
+
+
+
```
Comment (akash-1318) [inline documentation.html:992]: you are providing too much space between the lines use formatting or use prettier . Else everything looks good you can merge it.

File: documentation.html [hunk 4]
```diff
@@ -696,11 +696,274 @@ <h3>Rs.4900 <Span>Rs.9999</Span></h3>
...
+                    </p>
+
+                    <div class="display flex-row">
+                        
+                            <img class="square-img" src="./images/nike.png" alt="sneakers">
+                            <img class="square-img" src="./images/sample1.jpg" alt="alt-image">
+                            <img class="square-img" src="./images/sample2.jpg" alt="alt-image">
+
+
+                    </div>
```
Comment (tusharkandpal) [inline documentation.html]: format your code

File: documentation.html [hunk 5]
```diff
@@ -696,11 +696,274 @@ <h3>Rs.4900 <Span>Rs.9999</Span></h3>
...
+                        Note : Resize the browser window, to see the responsive property.
+                    </p>
+
+                    <div class="display flex-row">
+                        
+                            <img class="responsive-img" src="./images/nature.jpg" alt="sneakers">
+                            
+
+
+                    </div>
```
Comment (tusharkandpal) [inline documentation.html]: here too

**Review by tusharkandpal — APPROVED** [review]
please format your code ! else is good 👍🏻

File: css/input.css (hunk 1 above)
Comment (yogesh-) [reply css/input.css:26]: done.

File: documentation.html (hunk 2 above)
Comment (yogesh-) [reply documentation.html]: this is coming from code snippet tool so ignoring it.

File: documentation.html (hunk 4 above)
Comment (yogesh-) [reply documentation.html]: done

File: documentation.html (hunk 5 above)
Comment (yogesh-) [reply documentation.html]: done.

**Pull request merged**